### DISCUSSION TITLE
怎么设置双层导航栏

### DISCUSSION THREAD

**Discussion by susgit** [vuepress/core#969] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/68681019/175875928-e66ab9fb-a795-45ec-8762-94cdc471793b.png)
效果如图

**Comment by yzqdev** [comment, marked as answer]
you could design your own theme. here is how uniapp docs did.  [https://github.com/dcloudio/unidocs-zh](https://github.com/dcloudio/unidocs-zh/tree/master/docs/.vuepress/theme/components)
